### PR TITLE
Add option to skip clang-format execution

### PR DESCRIPTION
**Pull request by krypty**
Hello,

This option allows to use Format.cmake to only format CMake files (e.g. in the case C++ files formatting is handled by other means).

Please tell if I got any wrong with regards to contributing to this project.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 option(FORMAT_SKIP_CMAKE "Skip CMake formatting.")
+option(FORMAT_SKIP_CLANG "Skip CMake formatting.")
```
Comment (TheLartians) [inline CMakeLists.txt:2]: The description should be updated as well.

```suggestion
option(FORMAT_SKIP_CLANG "Skip Clang formatting.")
```

**Review by TheLartians — CHANGES_REQUESTED** [review]
Thanks for the PR and implementing this feature! One small thing: could you run cmake-format on this code itself?

File: CMakeLists.txt (hunk 1 above)
Comment (krypty) [reply CMakeLists.txt]: Fixed

**Review by TheLartians — APPROVED** [review]
Thanks for the changes!

**Pull request merged**